src/main/java/core/basesyntax/shop/service/impl/ShopDaoImpl.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.shop.service.impl;
+
+import core.basesyntax.shop.InsufficientGoodsException;
+import core.basesyntax.shop.db.FruitShopStorage;
+import core.basesyntax.shop.service.ShopDao;
+import java.util.Map;
+
+public class ShopDaoImpl implements ShopDao {
```

Comment:
It's not a service.
Seems should be FruitStorageDao and be putted to the separate package.